product-mini/platforms/windows/wasi_filtered_tests.json

```diff
@@ -1,34 +1,7 @@
 {
-    "WASI C tests": {
-        "fdopendir-with-access": "Not implemented",
-        "lseek": "Not implemented"
-    },
     "WASI Rust tests": {
-        "dangling_symlink": "Not implemented",
-        "directory_seek": "Not implemented",
-        "dir_fd_op_failures": "Not implemented",
-        "fd_advise": "Not implemented",
-        "fd_fdstat_set_rights": "Not implemented",
-        "fd_filestat_set": "Not implemented",
-        "fd_flags_set": "Not implemented",
-        "fd_readdir": "Not implemented",
-        "file_allocate": "Not implemented",
-        "file_seek_tell": "Not implemented",
-        "file_truncation": "Not implemented",
-        "nofollow_errors": "Not implemented",
-        "path_exists": "Not implemented",
-        "path_filestat": "Not implemented",
-        "path_link": "Not implemented",
-        "path_open_preopen": "Not implemented",
-        "path_rename": "Not implemented",
-        "path_rename_dir_trailing_slashes": "Not implemented",
-        "path_symlink_trailing_slashes": "Not implemented",
-        "poll_oneoff_stdio": "Not implemented",
-        "readlink": "Not implemented (path_symlink)",
-        "symlink_create": "Not implemented",
-        "symlink_filestat": "Not implemented",
-        "symlink_loop": "Not implemented",
-        "truncation_rights": "Not implemented"
+        "path_filestat": "This test attempts to open a file descriptor with fdflags APPEND & SYNC which is not supported on Windows",
```

Comment:
I'll raise a separate PR in wasi testsuite to amend the test behaviour - as far as I can tell APPEND isn't important to the test. The test just asserts that the flag is present if it was set so we can change it to another fdflag which doesn't have this issue on Windows.